### PR TITLE
V1 initial project foundation

### PR DESCRIPTION
**Pull request by sfranusic**
Clean up the iOS project structure and create the foundation for the Murcielago app

### CODE REVIEW

File: ios/Murcielago/Murcielago/UserInterface/Views/MainView/MainViewModel.swift [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        Task {
+            serviceActivity = true
+            clearErrorMessage()
+            async let result = authenticator.signIn(username: username, password: password)
+
+            if await result == false {
+                displayErrorMessage(type: .invalidCredentials)
+            }
+            await updateAuthentication()
+            serviceActivity = false
```
Comment (romanarr19191) [inline ios/Murcielago/Murcielago/UserInterface/Views/MainView/MainViewModel.swift:28]: Interesting, out of curiosity, why are you adding an authenticator? Is it necessary for this project?

File: ios/Murcielago/Murcielago/UserInterface/Views/SignInView/SignInView.swift [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+            .frame(width: 100)
+            .aspectRatio(1, contentMode: .fit)
+    }
+
+    private var loginUserInterface: some View {
+        VStack {
+            statusView
+            textFieldsView
+            signInButton
+
```
Comment (romanarr19191) [inline ios/Murcielago/Murcielago/UserInterface/Views/SignInView/SignInView.swift:40]: nit: extra space lol

**Review by romanarr19191 — COMMENTED** [review]
Nice! I like how organized your menu views are, and that you're not neglecting unit and UI tests!

File: ios/Murcielago/Murcielago/UserInterface/Views/MainView/MainViewModel.swift (hunk 1 above)
Comment (sfranusic) [reply ios/Murcielago/Murcielago/UserInterface/Views/MainView/MainViewModel.swift:28]: Just setting up a placeholder for now. Eventually will add a real authentication model.

**Pull request merged**